config/config.go

```diff
@@ -259,7 +264,7 @@ func (cfg *AggregateStatsConfig) Validate() error {
 // ServerConfig contains the API server configuration.
 type ServerConfig struct {
 	// ChainName is the name of the chain (e.g. mainnet/testnet/local).
```

Comment:
```suggestion
	// ChainName is the name of the chain (i.e. mainnet/testnet). Custom/local nets are not supported.
	// This is only used for the runtime status endpoint.
```

The second line is optional, but it mellows out the second sentence. I guess the second sentence is optional too ...

FYI, the only place this param is ever used is here: https://github.com/oasisprotocol/oasis-indexer/blob/dc3834f3132e79a9e9b07ce61d41b0dea6b81898/storage/client/client.go#L1333-L1333, which is then only used to list the compute nodes that support a particular runtime: https://github.com/oasisprotocol/oasis-indexer/blob/dc3834f3132e79a9e9b07ce61d41b0dea6b81898/storage/client/client.go#L1363-L1363

This should ideally hook into your `custom_chain` approach but since it lives on the API-server side (as opposed to the analysis side), we don't have access to that. But looks like we don't need that kind of generality here, at least for now.

Reply:
oh good point. if we later go about adding custom chain support for the api side, we could also consider moving the chain info up a level so that it's not analyzer-specific. then access that piece of configuration from the api too